docs/google-script-integration.md

```diff
@@ -0,0 +1,11 @@
+## Google Script Integration
+
```

Comment:
Can we link to this page from `docs/README.md`

Reply:
Done now.